### PR TITLE
DOCS HTTP Strict Transport Security

### PR DESCRIPTION
**Pull request by chillu**
See https://github.com/silverstripe/cwp-installer/issues/24
Requires https://github.com/silverstripe/cwp-core/pull/73

### CODE REVIEW

File: docs/en/01_Working_with_projects/10_Security.md [hunk 1]
```diff
@@ -139,6 +139,50 @@ for each domain. If you are unsure, contact the [CWP Service Desk](https://www.c
...
+be encrypted (via accessing on the `https://` protocol).
+
+CWP environments can always be accessed
+via SSL, either through agency-provided certificates, or free certificates
+provided through the Let's Encrypt service ([details](https://www.cwp.govt.nz/working-with-cwp/instance-management/ssl-certificates/)).
+CWP websites already enforce the `https://` protocol for authenticated requests,
+e.g. to `Security/*` and `admin/*`.
+
+Starting with CWP 2.4, new projects will
+automatically be configured to send HSTS headers, and redirect all requests to `https://`.
```
Comment (robbieaverill) [inline docs/en/01_Working_with_projects/10_Security.md]: ❤️

File: docs/en/05_Releases_and_changelogs/cwp_2.4.0.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# 2.4.0
```
Comment (robbieaverill) [inline docs/en/05_Releases_and_changelogs/cwp_2.4.0.md]: We should create this in the 2.4 branch. There's also a changelog template which Cow would use to create this file if it had done during a release process - you could use that as a base for this file. See https://github.com/silverstripe/cwp-recipe-kitchen-sink/blob/master/.changelog.md.twig

Comment (chillu) [reply docs/en/05_Releases_and_changelogs/cwp_2.4.0.md]: I've retargeted to 2.4, and used the "New features" header from the template. I'll leave the other templated bits to the CWP release managers, probably safer than copy pasting placeholders which then get missed later during the actual release.

**Comment by robbieaverill** [comment]
We need to remember to update the link to the release announcement

**Pull request merged**